### DISCUSSION TITLE
Number of aligned reads distinct between preset rna-seq and  generic-amplicon-with-umi

### DISCUSSION THREAD

**Discussion by AntonellaGuo** [milaboratory/mixcr#1650] [Q&A]
Hi there, 

I got a bulk rna-seq data of T cells with 10X Single Cell Mouse TCR Amplification Kit.  Although it's a weird way to build the library, I still would like to do the analysis. Since the library has 16bp 10x barcode and 10bp UMI in read 1,  I treated this 26bp as a unique UMI and  used the following setting
```
 mixcr analyze generic-amplicon-with-umi \
     --species mmu \
     --rna \
     --rigid-left-alignment-boundary \
     --floating-right-alignment-boundary C \
     --tag-pattern '^(UMI:N{26}) \^(R2:*)' \
        /data/Homo-1_S1_L001_R1_001_val_1.fq.gz \
        /data/Homo-1_S1_L001_R2_001_val_2.fq.gz \
        /data/Homo-1

```
And I got this report,  which shows that only 5.2% reads aligned.
_Total sequencing reads: 16382057
Successfully aligned reads: 784470 (4.79%)
Coverage (percent of successfully aligned):
  CDR3: 716849 (91.38%)
  FR3_TO_FR4: 103 (0.01%)
  CDR2_TO_FR4: 82 (0.01%)
  FR2_TO_FR4: 707 (0.09%)
  CDR1_TO_FR4: 590 (0.08%)
  VDJRegion: 222 (0.03%)
Alignment failed: no hits (not TCR/IG?): 10151902 (61.97%)
Alignment failed: absence of J hits: 5442259 (33.22%)
Alignment failed: absent barcode: 3426 (0.02%)_

Instead of using the `generic-amplicon-with-umi `, I also tried `rna-seq`
```
     mixcr analyze rna-seq --species mmu \
             --tag-pattern "^(UMI:N{10})\^(R2:*)" \
             /data/Homo-1_S1_L001_R1_001_val_1.fq.gz \
             /data/Homo-1_S1_L001_R2_001_val_2.fq.gz \
             /data/Homo-1
```
The report shows that 55.8% reads aligned.
_Total sequencing reads: 16382057
Successfully aligned reads: 9141188 (55.8%)
Coverage (percent of successfully aligned):
  CDR3: 719411 (7.87%)
  FR3_TO_FR4: 178 (0%)
  CDR2_TO_FR4: 908 (0.01%)
  FR2_TO_FR4: 407 (0%)
  CDR1_TO_FR4: 955 (0.01%)
  VDJRegion: 465 (0.01%)
Alignment failed: no hits (not TCR/IG?): 2071405 (12.55%)
Alignment failed: no CDR3 parts: 4813938 (29.18%)
Alignment failed: low total score: 406572 (2.46%)_

I would like to know why there is a big difference between the two results and to analyze this data which preset do you preferred.

Thx

**Comment by mizraelson** [comment]
Hi,

The difference you are seeing is because the `generic-amplicon-with-umi` preset filters out all reads that do not cover the full CDR3 region, unlike the `rna-seq` preset, which preserve reads if they cover at least part of the CDR3 for further assembly. The 10x Genomics protocol involves fragmentation, which is why the `rna-seq` preset shows a higher alignment rate.

It's hard to determine which preset would be best for this type of data. Did you mean that instead of using a cell suspension, you extracted bulk RNA from multiple cells and used this solution with the 10x Genomics microfluidics cartridge?
